site/components/map/ServicesMap.tsx

```diff
@@ -172,17 +180,52 @@ const Map = ({
                 const stop: Stop[] = getSelectedStopsFromMapMarkers(markerData, id);
                 const atcoCodes = getAtcoCodesFromSelectedStops(stop);
 
-                const servicesInPolygon = await fetchServicesByStops({
+                const servicesInPolygon: ServiceByStop[] = await fetchServicesByStops({
```

Comment:
Is the type needed here? Typescript should be able to use the return type of the function

Reply:
removed